ietf/ietfauth/views.py

```diff
@@ -744,17 +744,17 @@ def login(request, extra_context=None):
             logout(request)
             response = render(request, 'registration/missing_person.html')
         if require_consent:
-            messages.warning(request, mark_safe("""
+            messages.warning(request, mark_safe('''
 
                 You have personal information associated with your account which is not
                 derived from draft submissions or other ietf work, namely: %s.  Please go
-                to your <a href='/accounts/profile'>account profile</a> and review your
-                personal information, then scoll to the bottom and check the 'confirm'
+                to your <a href="/accounts/profile">account profile</a> and review your
+                personal information, then scoll to the bottom and check the "confirm"
```

Comment:
It doesn't - it replaces the single quotes with double quotes in the HTML block. If you prefer, I could escape the double quotes and leave the bracketing triple-quotes unchanged?

Reply:
Look at the double-quotes now around the word `confirm` in the text.
Reworking the way the value of the href attribute is quoted seems in scope for the pr.
changing quotes in the text between tags looks like an unintended overreach, and I wanted to call it out in case there were other places that might have unintentionally been changed.

This particular change to the text isn't harmful in and of itself, but it isn't what you intended, I suspect.

Reply:
Ah! I did a search/replace, which caught other quotes. Will undo.